homework13/js/main.js

```diff
@@ -0,0 +1,101 @@
+//Завдання 1. У інпут-полі користувач може ввести кількість відповідних секунд появи повідомлення
+
+const inputTime = document.querySelector('.write-number');
+const buttonTimeoutMessage = document.querySelector('.btn-timeout-message');
+
+const messageTimeout = () =>{
+    alert(`russia and russian have left till they disappear.`);
+}
+
+buttonTimeoutMessage.addEventListener('click', (event)=>{
+    event.preventDefault();
+    if(inputTime.value === ''){
+        alert('Поле не може бути пустим!');
+    } else if(inputTime.value.split('-')){
```

Comment:
This condition makes an issue


https://user-images.githubusercontent.com/34886027/199950819-59c09ef4-f6dd-4103-a45d-38885c5a4bb8.mov